src/psim/fc/orbit_controller.cpp

```diff
@@ -114,7 +116,18 @@ void OrbitController::step() {
     gnc::control_orbit(_orbit_controller, data, actuation);
 
     if (lin::all(lin::isfinite(actuation.J_ecef))) {
-      J_ecef = actuation.J_ecef;
+      //get random noise
+      Vector3 random_noise = lin::gaussians<Vector3>(_randoms);
+
+      // normalize the random noise
+      random_noise = random_noise / lin::norm(random_noise);
```

Comment:
don't normalize

Reply:
Lol yeah this is dumb, forgot the extra gaussian anyway